src/Updatedge.net.Tests/AvailabilityServiceTests.cs

```diff
@@ -0,0 +1,208 @@
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Updatedge.net.Entities.V1;
+using Updatedge.net.Entities.V1.Availability;
+using Updatedge.net.Services.V1;
+using Flurl.Http.Testing;
+using Updatedge.net.Exceptions;
+
+namespace Updatedge.net.Tests
+{
+    public class AvailabilityServiceTests
+    {
+        private HttpTest _httpTest;
+        private AvailabilityService _availabilityService;
+        private OkApiResult<List<WorkerAvailabilityIntervals>> _expectedOkResult;
+        private List<string> _userIdList;
+        
+        [SetUp]
+        public void Setup()
+        {
+            // Put Flurl into test mode
+            _httpTest = new HttpTest();
+
+            // instantiate the Availability service
+            _availabilityService = new AvailabilityService("https://localhost/", "1234567890");
```

Comment:
Can I suggest you look at AutoFixture for some of these dummy values?  It's a nice way of generating random structured data.  Not too important here, but useful to have in your armoury :)

https://github.com/AutoFixture/AutoFixture